### ISSUE TITLE
Cannot debug using tntdbg

### ISSUE THREAD

**Issue by mweichert** [trueinteractions/tint2#172]
Whenever I attempt to debug on Mac OS X, I get:

> No browser matching [chrome,chromium,opera] found in the system! If this is not true, submit a bug
> report on https://github.com/benderjs/browser-launcher2
> Please open the URL manually in Chrome/Chromium/Opera or similar browser
> Debugger listening on port 5858

I have browser-launcher3 installed, globally. If I run:

`node /usr/local/lib/node_modules/browser-launcher3/example/detect.js`

> `parseFile()` is deprecated. Use `parseString()` instead.
> `parseString()` is deprecated. It's not actually async. Use `parse()` instead.
> Available browsers:
> [ { name: 'chrome',
>    version: '47.0.2526.16',
>    type: 'chrome',
>    command: '/Applications/Google Chrome.app' },
>  { name: 'firefox',
>    version: '41.0.2',
>    type: 'firefox',
>    command: '/Applications/Firefox.app/Contents/MacOS/firefox-bin' },
>  { name: 'safari',
>    version: '9.0',
>    type: 'safari',
>    command: '/Applications/Safari.app' } ]

But I run using tint I get something different:

`tint /usr/local/lib/node_modules/browser-launcher3/example/detect.js`

> Available browsers:
> []

Any clue?

**Comment by trevorlinton** [comment]
@mweichert Our apologies, it appears in the move to node 4.1.2, upstream the gyp variable was moved from PLATFORM, to NODE_PLATFORM.  node v12 overwrote what the OS said to be "win32" and "darwin", however node 4.1.2 uses its own define.  This will be fixed in the next release of Tint (v2.2.2) due out Oct 26th.

**Comment by mweichert** [comment]
Thanks. Indeed, this seems to be fixed - as the browser window opens as it should. 

Unfortunately though, no scripts seem to be loaded in the Chrome browser window. 

I'm just doing a simple test at the moment:

```
require('Common');
var win = new Window(); // Create a new window.
application.name = 'HelloTint';
debugger;
win.visible = true; // make sure the window is shown.
```
No luck debugging yet.

**Comment by mweichert** [comment]
I just wanted to let you know that this was resolved in 2.2.2. Thanks!
